fhir-path/src/main/java/org/linuxforhealth/fhir/path/FHIRPathQuantityNode.java

```diff
@@ -32,6 +50,31 @@ protected FHIRPathQuantityNode(Builder builder) {
         quantityUnit = getQuantityUnit(quantity);
         quantityValue = getQuantityValue(quantity);
     }
+    
+    /**
+     * Method to return the QuantitySubType wrapped by this FHIRPathQuantityNode.
+     * @return the QuantitySubType, if the sub type wrapped by this FHIRPathQuantityNode is valid, null otherwise.
```

Comment:
```suggestion
     * @return the QuantitySubType wrapped by this FHIRPathQuantityNode (if there is one), otherwise null.
```

Reply:
This has been fixed. The java doc has been updated as suggested.